public/script.js

```diff
@@ -0,0 +1,106 @@
+// Selecteer het <html> element
+const htmlElement = document.documentElement;
+
+// Voeg de class "js" toe aan het <html> element
+htmlElement.classList.add("js");
+
+const checkbox = document.querySelector(".theme"); //theme button
+const body = document.body; //body
+const hero = document.querySelector("nav:nth-child(1) a img"); // nav logo
+
+const openBtn = document.querySelector(".openbtn"); // burgermenu button
+const mobilenav = document.querySelector(".wrapper-nav nav ul"); // open/close menu (for mobile)
+
+const houseButtons = document.querySelectorAll(".houseDetail button[id^='house']"); // rate button
+let starForms = document.querySelectorAll(".houseDetail form[id^='houseForm']"); // form section inside the li
+const formSubmit = document.querySelector(".wrapper-house ul .houseDetail form button[id^='houseSubmit']"); // submit rate button
+
+
+
+
+// changes funda logo
+document.addEventListener("DOMContentLoaded", (event) => {
+  checkbox.addEventListener("change", () => {
+    if (checkbox.checked) {
+      body.classList.add("alter");
+    } else {
+      body.classList.remove("alter");
+    }
+
+    if (checkbox.checked) {
+      // Check if the checkbox is checked
+      hero.src = "/images/01-funda-Orange.png"; // Set the image to dark mode
+    } else {
+      hero.src = "/images/02-funda-Blue.png"; // Set the image to light mode
+    }
+  });
+});
+
+//  burgermenu function
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
Hier ben je ook opnieuw een listener op 'DOMContentLoaded' aan het toevoegen, je kunt in principe gewoon de code van lijn 41-44 toevoegen aan de eerste listener op deze event vanaf lijn 22.

Zelfde geld voor de code van lijn 49-65 en 101-105 :)

Reply:
heb onnodige domcontenloaded events weg gehaald

hier is een voorbeeld
```javascript
// show form
houseButtons.forEach((button) => {
  button.addEventListener("click", () => {
    const houseId = button.id.replace("house", "houseForm");
    const starFormElement = document.getElementById(houseId);
    if (starFormElement) {
      starFormElement.classList.toggle("ShowForm");

      if (starFormElement.classList.contains("ShowForm")) {
        button.textContent = "Close";
        button.classList.add("rotate");
      } else {
        button.textContent = "Rate";
        button.classList.remove("rotate");
      }
    }
  });
});
```